test/token-service/test_tokenCreateTransaction.js

```diff
@@ -6293,26 +4723,21 @@ describe.only("TokenCreateTransaction", function () {
       assert.fail("Should throw an error");
     });
 
-    it("(#61) Creates a token with a fixed fee that is assessed with a deleted token", async function () {
-      let response = await JSONRPCRequest("createToken", {
+    it.skip("(#61) Creates a token with a fixed fee that is assessed with a deleted token", async function () {
```

Comment:
Another test skipped?

Reply:
This is related to the TokenCreateTransaction custom fees bugs in services. I updated the Notion file— I thought I had added it earlier, but it seems I hadn’t. I’ve highlighted it in yellow; once you review it, you can unhighlight it.

Reply:
Ah it involves a deleted token, we can look at this one again on the `deleteToken` branch once this gets merged